doc/modules/language-guide/pages/language-manual.adoc

```diff
@@ -1231,6 +1236,7 @@ The case sensitivity of file references depends on the host operating system so
 (Remark: when building multi-canister projects with the {sdk-short-name}, {proglang} programs can typically import canisters by alias (e.g. `import C "canister:counter"`), without specifying low-level canister ids (e.g. `import C "ic:lg264-qjkae"`).
```

Comment:
```suggestion
(Remark: when building multi-canister projects with the {sdk-short-name}, {proglang} programs can typically import canisters by alias (e.g. ``import C "canister:counter"``), without specifying low-level canister ids (e.g. ``import C "ic:lg264-qjkae"``.)
```

Reply:
Let's change the remark above too?

Reply:
3e8226ce041e941839e53d4803bdda4d8ce29298